spec/gaffe/errors_controller_resolver_spec.rb

```diff
@@ -42,6 +43,16 @@
         it { expect(controller).to eql :api_controller }
       end
 
+      context 'with error coming from matching HTTP_HOST' do
+        let(:env) do
+          request.env.merge(
+            'HTTP_HOST' => 'www.com',
+            'REQUEST_URI' => '/'
+          )
+        end
```

Comment:
Even if we merge two keys in the `env`, I think we should keep the `let` on a single line, like with the other contexts.

```ruby
let(:env) { request.env.merge 'HTTP_HOST' => 'www.com', 'REQUEST_URI' => '/' }
```